packages/core/src/request.ts

```diff
@@ -28,19 +28,24 @@ function enhanceEventWithSdkInfo(event: Event, sdkInfo?: SdkInfo): Event {
 }
 
 /** Creates a SentryRequest from a Session. */
-export function sessionToSentryRequest(session: Session, api: API): SentryRequest {
+export function sessionToSentryRequest(session: Session | SessionAggregates, api: API): SentryRequest {
   const sdkInfo = getSdkMetadataForEnvelopeHeader(api);
   const envelopeHeaders = JSON.stringify({
     sent_at: new Date().toISOString(),
     ...(sdkInfo && { sdk: sdkInfo }),
   });
+  let type: SentryRequestType = 'session';
+  if ('aggregates' in session) {
+    // I know this is hacky but we don't want to add `session` to request type since it's never rate limited
+    type = 'sessions' as SentryRequestType;
+  }
```

Comment:
```suggestion
  // I know this is hacky but we don't want to add `session` to request type since it's never rate limited
  let type: SentryRequestType = 'aggregates' in session ? 'sessions' as SentryRequestType : 'session';
```